package.json

```diff
@@ -22,26 +22,29 @@
     "build:webpack": "yarn build:browser && yarn build:examples-babel && ttsc --build examples/packages/webpack/tsconfig.json && yarn workspace @compiled/webpack-app build",
     "build:webpack:extract": "yarn build:browser && yarn build:examples-babel && ttsc --build examples/packages/webpack/tsconfig.json && cd examples/packages/webpack && EXTRACT_TO_CSS=true yarn build",
     "bundlesize": "yarn build && size-limit",
-    "clean": "rm -rf node_modules/.cache && find . -name *.tsbuildinfo -type f -delete",
+    "clean": "yarn clean:node-modules && yarn clean:ts-cache && yarn clean:dist",
+    "clean:dist": "find . -name \"dist\" -not -path \"*/node_modules/*\" -type d -exec rm -rf {} +",
+    "clean:node-modules": "rm -rf node_modules/.cache",
+    "clean:ts-cache": "find . -name \"*.tsbuildinfo\" -type f -delete",
     "flow-types": "scripts/flow-types.sh",
     "postinstall": "npx yarn-deduplicate && yarn --ignore-scripts",
     "lint": "eslint --config .eslintrc.js --ext flow,json,ts,tsx .",
     "lint:fix": "yarn lint -- --fix",
     "prettier:check": "prettier ./ --check",
     "prettier:fix": "prettier ./ --write",
     "release": "yarn clean && yarn build && yarn changeset publish",
-    "start": "npx nodemon --exec \"yarn build && start-storybook -p 6006 --ci\" --watch packages/babel-plugin/ -e tsx",
+    "start": "npx nodemon --exec \"start-storybook -p 6006 --ci\" --watch packages/babel-plugin/ -e tsx",
     "start:cli": "cd packages/cli && yarn start",
     "start:inspect": "npx nodemon --exec \"node --inspect-brk node_modules/.bin/start-storybook -p 6006 --ci\" --watch packages/babel-plugin/ -e tsx",
     "start:parcel": "yarn build:browser && yarn build:esm && ttsc --build examples/packages/parcel/tsconfig.json && yarn workspace @compiled/parcel-app start",
     "start:prod": "NODE_ENV=production yarn start",
     "start:ssr": "yarn build && cd examples/packages/ssr && yarn start",
     "start:webpack": "yarn build:browser && yarn build:examples-babel && ttsc --build examples/packages/webpack/tsconfig.json && yarn workspace @compiled/webpack-app start",
     "start:webpack:extract": "yarn build:browser && yarn build:examples-babel && ttsc --build examples/packages/webpack/tsconfig.json && cd examples/packages/webpack && EXTRACT_TO_CSS=true yarn start",
-    "test": "yarn build:cjs && yarn build:esm && jest --no-cache",
+    "test": "jest --no-cache",
```

Comment:
Removed the need for building when running tests! This means when you use any of Compiled's dependencies, anywhere in tests, you won't be stuck in that bizzaro world of needing to restart the test runner.

This doesn't help for the jest matcher or babel plugin though since the jest environment needs them to run - so we'll need to keep restarting them when developing them with the react package.

Example:

https://user-images.githubusercontent.com/6801309/142388009-7397c030-c563-47de-86ee-5d0cb394e4ac.mov